example/Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.5.1'
+gem 'rails', '~> 4.2'
 gem 'sqlite3'
 
 gem 'dotenv-rails'
-gem 'shopify_app', '~> 7.0.0'
+gem 'shopify_app', '~> 7.2.0'
```

Comment:
Yeah, I think this is fair. I've been playing with this all morning, and I think it can go. I tried out a few different versions and I think it's safe. (One aside though: because of SQLite, this can't be run on Heroku.)